### DISCUSSION TITLE
Benchmark speed between OpenAI and OpenLLM

### DISCUSSION THREAD

**Discussion by louis030195** [bentoml/OpenLLM#769] [Q&A]
Curious, are there benchmarks comparing speed between OpenAI and OpenLLM in terms of token/s?

**Comment by aarnphm** [comment]
I'm not sure if I understand the questions, but this is more like comparing apples to oranges. Are you comparing gpt4 against llama running with vllm?

If you want to compare pure tps then I don't have a number on my hands atm.
